sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -65,10 +66,11 @@ def __init__(self, base_url: str, credential: TokenCredential, **kwargs: Any) ->
 
         self._sync_token_policy = SyncTokenPolicy()
 
-        credential_scopes = kwargs.pop("credential_scopes", [f"{base_url.strip('/')}/.default"])
+        credential_scopes = kwargs.pop("credential_scopes", [get_default_scope(base_url)])
         # Ensure all scopes end with /.default
         kwargs["credential_scopes"] = [
-            scope if scope.endswith("/.default") else f"{scope}/.default" for scope in credential_scopes
+            scope if scope.endswith(_DEFAULT_SCOPE_SUFFIX) else f"{scope}{_DEFAULT_SCOPE_SUFFIX}"
```

Comment:
Should we trim potential trailing `/` on the scopes before appending the suffix?